packages/doctor-sdk/tsconfig.json

```diff
@@ -1,6 +1,11 @@
 {
   "extends": "@rsbuild/tsconfig/base",
   "compilerOptions": {
+    "experimentalDecorators": true,
```

Comment:
Are we using decorators? We should avoid using it because decorator is not stable yet.

Reply:
yes，Decorator